### PR TITLE
Revert back to GRUB 2.02.

### PR DESCRIPTION
**Pull request by kacf**
GRUB 2.04 has trouble booting on Beaglebone.

### CODE REVIEW

**Comment by kacf** [comment]
@lluiscampos: I still need to test this on the BBB tomorrow.

File: grub-efi/docker-create-grub-efi-binaries [hunk 1]
```diff
@@ -29,12 +29,10 @@ for arch in x86-64 arm aarch64; do
     echo -n "Extracting binaries for $arch..."
     docker run --rm --entrypoint "/bin/sh" \
         -v $(pwd)/output/$arch:/export grub-efi:$arch \
-        -c "cp -r /install/* /export"
+        -c "chown -R --reference /export /install/* && mv /install/* /export"
     echo " done"
 done
 
-sudo chown -R $(id -u):$(id -g) output/*
-
```
Comment (lluiscampos) [inline grub-efi/docker-create-grub-efi-binaries]: Nice trick! Thanks.

Please remove it also from [the other script](https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/docker-create-integration-binaries#L29) as well as for the install dependencies in GitLab Pipeline ([here](https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/.gitlab-ci.yml#L25) and [here](https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/.gitlab-ci.yml#L39))

Comment (kacf) [reply grub-efi/docker-create-grub-efi-binaries]: Done!

**Comment by kacf** [comment]
I also enabled parallel make. Got tired of waiting. ;-)

**Comment by lluiscampos** [comment]
Remember to update now `mender-convert` to use GRUB 2.02 version

**Pull request merged**